src/services/ContactRepository.ts

```diff
@@ -63,8 +62,16 @@ class ContactRepository {
       })
       .then(response => response.json())
       .then((contact: Contact) => {
+        const dID = contact.did
+
         PersistentStore.store.dispatch(Contacts.Store.addContact(contact))
-        return contact
+
+        return {
```

Comment:
Why is this needed, since `contact` already has a `did` property?

Reply:
It doesn't or at least at some point it is lost.

Reply:
Yes, I have fixed it.